app/src/main/java/com/example/appetite/MainActivity.java

```diff
@@ -120,66 +107,66 @@ private void initRecyclerView() {
     }
 
     private void setRecyclerViewData(List<Feature> features, Point deviceLocation) {
-        // Nearby Restaurant Daten Felder initialisieren
+        // initialize ArrayList of nearby restaurants
         List<NearbyRestaurants> restaurantsDataList = new ArrayList<>();
-        // Platzhalterdaten
+        // placeholder data
```

Comment:
```suggestion
```

Reply:
this comment is no longer required because we're no longer using placeholder data but retrieve it from the api. Before that I just initialized it with some random data.